// gsap/all.js
import gsap from "gsap/gsap-core.js";
import CSSPlugin from "gsap/CSSPlugin.js";
var gsapWithCSS = gsap.registerPlugin(CSSPlugin) || gsap,
    // to protect from tree shaking
TweenMaxWithCSS = gsapWithCSS.core.Tween;
export { gsapWithCSS as gsap, gsapWithCSS as default, TweenMaxWithCSS as TweenMax, CSSPlugin };
export { TweenLite, TimelineMax, TimelineLite, Power0, Power1, Power2, Power3, Power4, Linear, Quad, Cubic, Quart, Quint, Strong, Elastic, Back, SteppedEase, Bounce, Sine, Expo, Circ, wrap, wrapYoyo, distribute, random, snap, normalize, getUnit, clamp, splitColor, toArray, mapRange, pipe, unitize, interpolate, shuffle, selector } from "gsap/gsap-core.js";
export * from "gsap/CustomEase.js";
export * from "gsap/Draggable.js";
export * from "gsap/CSSRulePlugin.js";
export * from "gsap/EaselPlugin.js";
export * from "gsap/EasePack.js";
export * from "gsap/Flip.js";
export * from "gsap/MotionPathPlugin.js";
export * from "gsap/Observer.js";
export * from "gsap/PixiPlugin.js";
export * from "gsap/ScrollToPlugin.js";
export * from "gsap/ScrollTrigger.js";
export * from "gsap/TextPlugin.js"; //BONUS EXPORTS
// export * from "./DrawSVGPlugin.js";
// export * from "./Physics2DPlugin.js";
// export * from "./PhysicsPropsPlugin.js";
// export * from "./ScrambleTextPlugin.js";
// export * from "./CustomBounce.js";
// export * from "./CustomWiggle.js";
// export * from "./GSDevTools.js";
// export * from "./InertiaPlugin.js";
// export * from "./MorphSVGPlugin.js";
// export * from "./MotionPathHelper.js";
// export * from "./ScrollSmoother.js";
// export * from "./SplitText.js";
